python_scripts/linear_models_ex_03.py

```diff
@@ -29,17 +29,17 @@
 data.head()
 
 # %% [markdown]
-# Now this is your turn to train a linear regression model on this dataset.
+# Now it is your turn to train a linear regression model on this dataset.
 # You will need to:
 # * create a linear regression model;
 # * execute a cross-validation with 10 folds and use the mean absolute error
-#   (MAE) as metric. Ensure to return the fitted estimators;
-# * compute mean and std of the MAE in thousands of dollars (k$);
-# * show the values of the coefficients for each feature using a boxplot by
-#   inspecting the fitted model returned from the cross-validation. Hint: you
+#   (MAE) as metric. Be sure to *return* the fitted *estimators*;
+# * inspect the fitted model using a box plot to show the distribution of values
+#   for the coefficients returned from the cross-validation. Hint:
 #   use the function
 #   [`df.plot.box()`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.box.html)
-#   to plot a box plot.
+#   to create a box plot.
+# * compute mean and std of the MAE in thousands of dollars (k$);
```

Comment:
Honestly I think we should stop using `;` in bullet points ...

If you want to follow @glemaitre on this one the previous bullet point should have a `;` and the final one a `.`

This is a good occasion for me to quote something I read recently (completely unrelated to the MOOC) that made me laugh:

> Guerre au point-virgule ! Ce parasite, ce timoré, cet affadisseur qui ne marque que l'incertitude, le manque d'audace, le flou de la pensée, et colle aux dents du lecteur comme du caramel mou !

The original quote seems to be from François Cavanna in Mignonne allons voir si la rose ...

Reply:
Thanks! Fixed it.